### ISSUE TITLE
Tools to Track & Monitor PostHog Usage

### ISSUE THREAD

**Issue by dknowles-vsk** [PostHog/posthog#5451]
## Is your feature request related to a problem?
We don't have an easy way to gather and visualize usage statistics for our PostHog deployment

## Describe the solution you'd like
We want usage statistics and visualizations on our PostHog deployment. Ideally, it would be great to run PostHog on itself. 

## Describe alternatives you've considered
Manually surveying teams as to whether they still use PostHog, which is time consuming and inaccurate.

## Additional context
We have an increasing number of teams trained to use PostHog. The goal is that these teams will go back to PostHog regularly to monitor their product areas. We need to understand whether these teams are actually doing this, or if they drop off after training ends.

**Comment by paolodamico** [comment]
CC @marcushyett-ph as we start thinking more on collab tools.

**Comment by marcushyett-ph** [comment]
cc: @timgl this is the one I was talking about, I wonder if there's an easy way to get key stats into a separate project..

I would imagine key metrics are going to be:

- Weekly active users
- Daily active users
- Weekly recurring retention on "insight viewed"

@dknowles-vsk any other key metrics you'd expect to see.

**Comment by dknowles-vsk** [comment]
My goal is to understand if people are getting useful information from PostHog and thus regularly using it. This requires me to assess the following:
- User retention and stickiness/habit (how often did a user return to the dashboard/insight? Are they returning daily or weekly or monthly or never?)
- Daily/weekly/monthly users (note that our sprints are fortnightly, which means weekly may not capture what we need)
- Feature usage (e.g. do 99% of our users use Funnels, or is it 50/50 user paths and trends?)
- Heatmaps or funnels (what are people clicking on and exploring but not able to use successfully)

In other words, if you could manage to run PostHog on a PostHog deployment I'd be happy 😉

**Comment by marcushyett-ph** [comment]
To add extra context here from a privacy compliance point of view:

The solution for this problem could help self-hosted users with compliance obligations, e.g. it could be used as an audit trail to understand who did what on PostHog and when - assisting with any investigations following an incident or otherwise.
